packages/renovate-config-cozy-konnector/package.json

```diff
@@ -6,22 +6,11 @@
   "renovate-config": {
     "default": {
       "extends": [
-        "config:base"
+        "cozy"
       ],
-      "timezone": "Europe/Paris",
       "minor": {
         "automerge": true
-      },
-      "packageRules": [
-        {
-          "excludePackageNames": [
-            "cozy-konnector-libs",
-            "cozy-app-publish"
-          ],
-          "schedule": "before 5am every monday"
-        }
-      ],
-      "prHourlyLimit": 0
```

Comment:
[0 is the fault value](https://renovatebot.com/docs/configuration-options/#prhourlylimit), so I don't think we need this?

Reply:
I suppose you are right